Stratis.Bitcoin/Features/BlockStore/BlockStoreLoopStep.cs

```diff
@@ -6,15 +6,36 @@
 
 namespace Stratis.Bitcoin.Features.BlockStore
 {
+    /// <summary>
+    /// The chain of block store loop steps that is executed when the
+    /// BlockStoreLoop's DownloadAndStoreBlocks is called
+    /// <seealso cref="BlockStoreLoop.DownloadAndStoreBlocks"/>
+    /// </summary>
     internal sealed class BlockStoreStepChain
     {
         private List<BlockStoreLoopStep> steps = new List<BlockStoreLoopStep>();
 
+        /// <summary>
+        /// Sets the next to execute in the chain
```

Comment:
...the next **step** to execute..

and shouldn't it be "in the loop" instead of "in the chain"?